bot/skills/towel_mode.py

```diff
@@ -60,14 +63,16 @@ def quarantine_user(user: User, chat_id: str, context: CallbackContext) -> None:
     minute = datetime.datetime.utcnow().minute
     quarantine[user.id] = minute
     quarantine_min[minute].append(user.id)
+
+    markup = InlineKeyboardMarkup([
+        [InlineKeyboardButton("Я не бот!", callback_data=42)]])
```

Comment:
@C-Pro nice magic number! 🎱🎱🎱

Reply:
For your total satisfaction I moved it to properly-named constant